src/Operators/finitedifference.jl

```diff
@@ -2073,24 +2073,20 @@ Base.Broadcast._broadcast_getindex_eltype(
 ) = eltype(bc)
 
 # check that inferred output field space is equal to dest field space
-@noinline inferred_stencil_spaces_error(
-    dest_space_type::Type,
-    result_space_type::Type,
-) = error(
-    "dest space `$dest_space_type` is not the same instance as the inferred broadcasted result space `$result_space_type`",
+inferred_stencil_spaces_error(::Type{A}, ::Type{B}) where {A, B} = error(
```

Comment:
can we mark this as `@noinline` to outline the error call?

Reply:
I’ll try this today